source/AI.cpp

```diff
@@ -4929,14 +4901,10 @@ void AI::IssueOrders(const Orders &newOrders, const string &description)
 		Messages::Add(who + description, Messages::Importance::High);
 	else
 	{
-		// Clear all the orders for these ships.
 		if(!isValidTarget)
 			Messages::Add(who + "unable to and no longer " + description, Messages::Importance::High);
 		else
 			Messages::Add(who + "no longer " + description, Messages::Importance::High);
-
-		for(const Ship *ship : ships)
-			orders.erase(ship);
```

Comment:
I removed this because rescinding one order doesn't necessarily mean that there are no orders remaining, but we probably still want to erase orders where there are no active orders set